trunk/SUAVE/Analyses/Stability/Fidelity_Zero.py

```diff
@@ -182,65 +167,77 @@ def __call__(self,conditions):
             # Derivative of yawing moment with respect to the rate of yaw
             cDw = aero.drag_breakdown.parasite['main_wing'].parasite_drag_coefficient # Might not be the correct value
             l_v = geometry.wings['vertical_stabilizer'].origin[0] + geometry.wings['vertical_stabilizer'].aerodynamic_center[0] - geometry.wings['main_wing'].origin[0] - geometry.wings['main_wing'].aerodynamic_center[0]
-            dynamic_stability.cn_r = Supporting_Functions.cn_r(cDw, geometry.wings['vertical_stabilizer'].areas.reference, Sref, l_v, span, geometry.wings['vertical_stabilizer'].dynamic_pressure_ratio, geometry.wings['vertical_stabilizer'].CL_alpha)
+            stability.static.Cn_r = Supporting_Functions.cn_r(cDw, geometry.wings['vertical_stabilizer'].areas.reference, Sref, l_v, span, geometry.wings['vertical_stabilizer'].dynamic_pressure_ratio, geometry.wings['vertical_stabilizer'].CL_alpha)
 
             # Derivative of rolling moment with respect to roll rate
-            dynamic_stability.cl_p = Supporting_Functions.cl_p(conditions.lift_curve_slope, geometry)
+            stability.static.Cl_p = Supporting_Functions.cl_p(conditions.lift_curve_slope, geometry)
 
             # Derivative of roll rate with respect to sideslip (dihedral effect)
             if 'dihedral' in geometry.wings['main_wing']:
-                dynamic_stability.cl_beta = Supporting_Functions.cl_beta(geometry, dynamic_stability.cl_p)
+                stability.static.Cl_beta = Supporting_Functions.cl_beta(geometry, stability.static.Cl_p)
             else:
-                dynamic_stability.cl_beta = np.zeros(1)
+                stability.static.Cl_beta = np.zeros(1)
 
-            dynamic_stability.cy_beta = 0
+            stability.static.Cy_beta = 0
 
             # Derivative of pitching moment with respect to pitch rate
             l_t                    = geometry.wings['horizontal_stabilizer'].origin[0] + geometry.wings['horizontal_stabilizer'].aerodynamic_center[0] - geometry.wings['main_wing'].origin[0] - geometry.wings['main_wing'].aerodynamic_center[0] #Need to check this is the length of the horizontal tail moment arm       
-            dynamic_stability.cm_q = Supporting_Functions.cm_q(conditions.lift_curve_slope, l_t,mac) # Need to check Cm_i versus Cm_alpha
+            stability.static.Cm_q  = Supporting_Functions.cm_q(conditions.lift_curve_slope, l_t,mac) # Need to check Cm_i versus Cm_alpha
```

Comment:
I know you didn't originally code this, but you should be creative in using aliases to shorten names for unpacking and packing

Reply:
Got it. The reason for this change was so I could store both static and dynamic stability into one data structure